src/main/java/com/example/spring/services/BookServiceImpl.java

```diff
@@ -35,4 +35,22 @@ public BookDto findById(Long id) {
                 () -> new EntityNotFoundException("Can't find element by id: " + id));
         return bookMapper.toDto(book);
     }
+
+    @Override
+    public void deleteById(Long id) {
+        bookRepository.deleteById(id);
+    }
+
+    @Override
+    public BookDto update(Long id, CreateBookRequestDto newBookDto) {
+        Book existingBook = bookRepository.findById(id).orElseThrow(
+                () -> new EntityNotFoundException("There's not book with id: " + id));
+        existingBook.setTitle(newBookDto.getTitle());
+        existingBook.setAuthor(newBookDto.getAuthor());
+        existingBook.setIsbn(newBookDto.getIsbn());
+        existingBook.setPrice(newBookDto.getPrice());
+        existingBook.setDescription(newBookDto.getDescription());
+        existingBook.setCoverImage(newBookDto.getCoverImage());
```

Comment:
```suggestion
        bookMapper.updateBookFromDto(bookDto, book);
```
use mapper for this logic